sdk/spring/scripts/replace_util.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+import argparse
+import os
+import time
+
+import in_place
+
+from log import log, Log
+
+config = {
+    'cosmos': {
+        'sdk/cosmos/azure-spring-data-cosmos-test/pom.xml': {
+            'artifact_id_tuple': (
+                ('azure-spring-data-2-3-cosmos', 'azure-spring-data-2-2-cosmos'),
+            )
+        }
+    },
+    'spring': {
+        'sdk/spring/azure-spring-boot-test-cosmosdb/pom.xml': {
+            'artifact_id_tuple': (
+                ('azure-cosmosdb-spring-boot-2-3-starter', 'azure-cosmosdb-spring-boot-2-2-starter'),
+            ),
+            'version_tuple': (
+                ('org.springframework.boot:spring-boot-starter-web', '2.2.9.RELEASE'),
+                ('org.springframework.boot:spring-boot-starter-actuator', '2.2.9.RELEASE'),
+                ('org.springframework.boot:spring-boot-starter-test', '2.2.9.RELEASE')
+            )
+        }
+    }
+}
+
+
+def main():
+    start_time = time.time()
+    change_to_root_dir()
+    log.debug('Current working directory = {}.'.format(os.getcwd()))
+    args = get_args()
+    init_log(args)
+    replace(args.module)
+    elapsed_time = time.time() - start_time
+    log.info('elapsed_time = {}'.format(elapsed_time))
+
+
+def change_to_root_dir():
+    os.chdir(os.path.dirname(os.path.realpath(__file__)))
+    os.chdir('../../..')
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        description = 'Replace artifact id in pom file.'
+    )
+    parser.add_argument(
+        '--module',
+        type = str,
+        choices = ['spring', 'cosmos'],
+        required = False,
+        default = 'cosmos',
+        help = 'Specify the target module.'
+    )
+    parser.add_argument(
+        '--log',
+        type = str,
+        choices = ['debug', 'info', 'warn', 'error', 'none'],
+        required = False,
+        default = 'info',
+        help = 'Set log level.'
+    )
+    parser.add_argument(
+        '--color',
+        type = str,
+        choices = ['true', 'false'],
+        required = False,
+        default = 'true',
+        help = 'Whether need colorful log.'
+    )
+    return parser.parse_args()
+
+
+def init_log(args):
+    log_dict = {
+        'debug': Log.DEBUG,
+        'info': Log.INFO,
+        'warn': Log.WARN,
+        'error': Log.ERROR,
+        'none': Log.NONE
+    }
+    log.set_log_level(log_dict[args.log])
+    color_dict = {
+        'true': True,
+        'false': False
+    }
+    log.set_color(color_dict[args.color])
+
+
+def replace(module):
+    """
+    Replace action
+    :param module:
+    :return:
```

Comment:
Remote the doc if there is no detail information.

Reply:
return void?

Reply:
I mean, we should delete this line, or add something after `return`.
Otherwise code reader may confused: does code writer forgot to write doc for the return value?